### ISSUE TITLE
[Filter Bar] changing the order of filters in filter bar component

### ISSUE THREAD

**Issue by rakesh4ui347** [SAP/ui5-webcomponents-react#5243]
### Is your feature request related to a problem?

This feature requires compliance with Fiori guidance. when we click on the Adapt filter button on the filter bar, it opens the filter dialog, then in the content section, we can see the filter's name, but we are unable to move it up or down. if you see the content on the filter dialog, the first column is  "field", but it should be "Filter".

### Describe the solution you'd like
https://sapui5.hana.ondemand.com/#/entity/sap.ui.comp.filterbar.FilterBar/sample/sap.ui.comp.sample.filterbar.DynamicPageListReport
Basically we move filter names up and down depending on our choice.
![image](https://github.com/SAP/ui5-webcomponents-react/assets/39724733/f3072b4f-5f73-448e-840b-5b04b423a4d0)
![image](https://github.com/SAP/ui5-webcomponents-react/assets/39724733/24bd031d-7457-4423-8f57-c9a3a48f1358)

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @rakesh4ui347 

it is now possible to change the order of filters inside the filters dialog by setting `enableReordering` on the `FilterBar` and applying a unique `orderId` to each `FilterGroupItem`. You can find an example of this [here](https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-filterbar--docs#filterbar-with-reordering).

Reordering filters by drag and drop is not supported, since the underlying `Table` component doesn't support it yet. (See [here](https://github.com/SAP/ui5-webcomponents/issues/7240))
